apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-item-presenter.tsx

```diff
@@ -1,18 +1,21 @@
-import { IMultilingualTextItem } from '@coscrad/api-interfaces';
+import { LanguageCode, MultilingualTextItemRole } from '@coscrad/api-interfaces';
 import { Language as LanguageIcon } from '@mui/icons-material';
 import { IconButton, Tooltip, Typography } from '@mui/material';
 import { getLabelForLanguage } from './text-presenters/get-label-for-language';
 
 interface MultilingualTextItemPresenterProps {
     isHeading: boolean;
-    item: IMultilingualTextItem;
+    text: string;
+    languageCode: LanguageCode;
+    role: MultilingualTextItemRole;
 }
 
 export const MultilingualTextItemPresenter = ({
     isHeading,
-    item,
+    text,
+    languageCode,
+    role,
 }: MultilingualTextItemPresenterProps): JSX.Element => {
-    const { text, languageCode, role } = item;
     return (
         <Typography variant={isHeading ? 'h4' : 'body1'} margin={'auto 0'}>
```

Comment:
I really don't like `isHeading` here. We should expose the variant (or a subtype) as the prop instead.